packages/web-components/src/wallet-connection/walletConnection.js

```diff
@@ -5,7 +5,16 @@ import { makeInteractiveSigner } from './makeInteractiveSigner.js';
 import { watchWallet } from './watchWallet.js';
 import { Errors } from '../errors.js';
 
-export const makeAgoricWalletConnection = async (chainStorageWatcher, rpc) => {
+/**
+ * @param {any} chainStorageWatcher
+ * @param {string} rpc
+ * @param {((error: unknown) => void)=} onRpcError
```

Comment:
not worth another push, but in agoric-sdk we use jsdoc syntax,
```suggestion
 * @param {((error: unknown) => void)} [onRpcError]
```